README.md

```diff
@@ -38,4 +39,27 @@ $tax = new TaxonomyBuilder( 'product-cat' );
 $tax->set_post_types( 'product' )
   ->set_labels( 'Product Category', 'Product Categories' )
   ->register();
+
+// Clean transients.
+$transient_cleaner = TransientCleaner::get_instance();
+$transient_cleaner->set_config([
```

Comment:
**question:** Could/should the `set_config` method be static? 

We could be more concise when configuring the tool: 

```php
TransientCleaner::set_config(array( /* ... */ ));
```

Reply:
@titouanmathis `set_config` use `$this` context, so it can't be static.